testsuite/interaction_potentials/dihedral_harmonic/test_dihedralharmonic.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import sys
+import time
+import math
+import espressopp
+import mpi4py.MPI as MPI
+
+import unittest
+
+def vec_pbc(u,v,cell):
+   #vector u-v
+   dx = u[0] - v[0]
+   dy = u[1] - v[1]
+   dz = u[2] - v[2]
+   dx = dx - round(dx/cell[0])*cell[0]
+   dy = dy - round(dy/cell[1])*cell[1] 
+   dz = dz - round(dz/cell[2])*cell[2] 
+   return dx,dy,dz
+
+def abslen(u):
+    return math.sqrt(u[0]*u[0]+u[1]*u[1]+u[2]*u[2])
+
+def cross(u,v):
+    c=[0.0,0.0,0.0]
+    c[0]=u[1]*v[2]-u[2]*v[1]
+    c[1]=u[2]*v[0]-u[0]*v[2]
+    c[2]=u[0]*v[1]-u[1]*v[0]
+    return c
+
+def dot(u,v):
+    c=u[0]*v[0]+u[1]*v[1]+u[2]*v[2]
+    return c
+
+def calc_dihedral(self,quadrupleslist):
+    #quadrupleslist contains [i,j,k,n]
```

Comment:
Yes, as you say below, I was using the function calc_dihedral to output phi and independently check computeEnergy, and just generally check that the system was behaving as expected (by printing phi as a function of time over many steps). I forgot to delete it from the unit test before pushing. 

Actually now I'm reluctant to delete it because I spent some time checking that all definitions of phi were consistent (IUPAC convention, function calc_dihedral, convention used by espressopp in various places, convention used by vmd, convention used in gromacs, convention used in dlpoly...) and this can probably be useful for someone else in the future for debugging some other dihedral function. (You can see I also used it in the gist https://gist.github.com/anonymous/82930269c0f47f3088b26d015cea8947.)

I'll leave it in and add another assertAlmostEqual which uses it.